### PR TITLE
Release Notes M.D.S v3.1.3

### PR DESCRIPTION
**Pull request by Kaur-Parminder**
Release Notes M.D.S v3.1.3

### CODE REVIEW

**Review by DavoudEshtehari — APPROVED** [review]

**Review by David-Engel — APPROVED** [review]

File: release-notes/3.1/3.1.3.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- System.Security.Principal.Windows 4.7.0 
+- System.Text.Encoding.CodePages 4.7.0 
+- System.Text.Encodings.Web 4.7.2
+- System.Diagnostics.DiagnosticSource 4.7.0 
+- System.Configuration.ConfigurationManager 4.7.0 
+- System.Runtime.Caching 4.7.0 
+- Azure.Identity 1.3.0
+- Microsoft.Identity.Client 4.14.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 5.6.0
+- Microsoft.IdentityModel.JsonWebTokens 5.6.0
```
Comment (lcheunglci) [inline release-notes/3.1/3.1.3.md:43]:
```suggestion
- Microsoft.Identity.Client 4.22.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.8.0
- Microsoft.IdentityModel.JsonWebTokens 6.8.0
```

File: release-notes/3.1/3.1.3.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Dependencies
+
+#### .NET Framework 4.6.1
+
+- Microsoft.Data.SqlClient.SNI 3.0.1
+- Azure.Identity 1.3.0
+- Microsoft.Identity.Client 4.14.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 5.6.0
+- Microsoft.IdentityModel.JsonWebTokens 5.6.0
```
Comment (lcheunglci) [inline release-notes/3.1/3.1.3.md:26]:
```suggestion
- Microsoft.Identity.Client 4.22.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.8.0
- Microsoft.IdentityModel.JsonWebTokens 6.8.0
```

File: release-notes/3.1/3.1.3.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- System.Buffers 4.5.1
+- System.Memory 4.5.4
+- System.Security.Principal.Windows 4.7.0
+- System.Text.Encoding.CodePages 4.7.0
+- System.Text.Encodings.Web 4.7.2
+- System.Runtime.Caching 4.7.0
+- Azure.Identity 1.3.0
+- Microsoft.Identity.Client 4.14.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 5.6.0
+- Microsoft.IdentityModel.JsonWebTokens 5.6.0
```
Comment (lcheunglci) [inline release-notes/3.1/3.1.3.md:73]:
```suggestion
- Microsoft.Identity.Client 4.22.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.8.0
- Microsoft.IdentityModel.JsonWebTokens 6.8.0
```

File: release-notes/3.1/3.1.3.md (hunk 2 above)
Comment (lcheunglci) [reply release-notes/3.1/3.1.3.md:26]: Looks like there's updates according to the nuget package explorer perhaps from the previous release.

File: release-notes/3.1/3.1.3.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- System.Security.Principal.Windows 4.7.0 
+- System.Text.Encoding.CodePages 4.7.0 
+- System.Text.Encodings.Web 4.7.2
+- System.Diagnostics.DiagnosticSource 4.7.0 
+- System.Configuration.ConfigurationManager 4.7.0 
+- System.Runtime.Caching 4.7.0 
+- Azure.Identity 1.3.0
+- Microsoft.Identity.Client 4.14.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 5.6.0
+- Microsoft.IdentityModel.JsonWebTokens 5.6.0
```
Comment (lcheunglci) [inline release-notes/3.1/3.1.3.md:58]:
```suggestion
- Microsoft.Identity.Client 4.22.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.8.0
- Microsoft.IdentityModel.JsonWebTokens 6.8.0
```

**Review by lcheunglci — APPROVED** [review]

**Pull request merged**